scripts/generator/graphql_generator/csharp/SDK/ConnectionLoader.cs.erb

```diff
@@ -9,7 +9,7 @@ namespace <%= namespace %>.SDK {
     public delegate object GetConnectionFromResponseDelegate(object response);
     public delegate void BuildQueryOnNodeDelegate(QueryRootQuery query, List<ConnectionQueryInfo> connectionInfosToBuildQuery, string nodeId, string alias);
     public delegate void BuildQueryOnEdgesNodeDelegate(object node, string after);
-    public delegate void ResponseNodeHandler(List<Node> nodes, List<string> errors, string httpError);
+    public delegate void ResponseNodeHandler(List<Node> nodes, ShopifyError error);
```

Comment:
I wasn't sure if we should only be generating the `ShopifyError` from the `ShopifyBuy` layer only versus having it here as well.